packages/webmachine/webmachine.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+version: "0.1.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+homepage: "https://github.com/inhabitedtype/ocaml-webmachine"
+dev-repo: "https://github.com/inhabitedtype/ocaml-webmachine.git"
+bug-reports: "https://github.com/inhabitedtype/ocaml-webmachine/issues"
+authors: ["Inhabited Type LLC"]
+license: "PROP"
+
+available: [
+  ocaml-version >= "4.01"
+]
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "webmachine"]
+]
+
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make]
+  [make "test"]
+]
+
+depends: [
+  "ounit" {test}
+  "ocamlfind" {build}
+  "cohttp"
```

Comment:
It does indeed need a cohttp constraint. Some earlier OCaml compiler versions only have 0.9.x available to them, which is why I added the `ocaml-version` constraint. Any idea what a safe lower bound would be? If not, I'll look into it for the next release and update this one at that time.